nuimo/src/main/kotlin/com/senic/nuimo/NuimoBluetoothController.kt

```diff
@@ -26,6 +26,9 @@ class NuimoBluetoothController(bluetoothDevice: BluetoothDevice, context: Contex
     private val mainHandler = Handler(Looper.getMainLooper())
     private var writeQueue = WriteQueue()
     private var matrixWriter: LedMatrixWriter? = null
+    private var rebootToDfuModeCharacteristic: BluetoothGattCharacteristic? = null
```

Comment:
Please make this a computed var that gets its value from `gatt?.characteristicWithUuid` by using gatt's proper API instead of `characteristicWithUuid` (that I just made up)